### PR TITLE
CR-1257 | tidying up the logging

### PR DESCRIPTION
**Pull request by akritisinhaa**
This ticket involved reviewing the logging in RHUI. There were some points raised regarding it and was discussed and agreed with Paul and the changes have been made accordingly.

For more information please see the ticket https://collaborate2.ons.gov.uk/jira/browse/CR-1257. 
To know more about the context of the changes please look at the **Questions** tab in the spreadsheet attached to this ticket and Paul's response on the ticket.

### CODE REVIEW

File: app/start_handlers.py [hunk 1]
```diff
@@ -99,6 +99,7 @@ async def get(self, request):
                     'page_url': View.gen_page_url(request)
                 }
         except KeyError:
+            logger.info('adlocation error')
```
Comment (Paul-Joel) [inline app/start_handlers.py:102]: Log text should be 'no adlocation present' - the KeyError happens when there is no adlocation provided, which is the normal behaviour

File: app/start_handlers.py [hunk 2]
```diff
@@ -146,10 +147,9 @@ async def post(self, request):
                     },
                     status=401)
             else:
+                logger.error('invalid access code', client_ip=request['client_ip'])
```
Comment (Paul-Joel) [inline app/start_handlers.py:150]: Log text needs to be something like 'error processing access code' - this error is triggered when RHSvc returns any error not triggered by an invalid code

**Review by Paul-Joel — CHANGES_REQUESTED** [review]
Couple of changes to the text of the messages

**Review by Paul-Joel — APPROVED** [review]
LGTM

**Review by rpcatling — APPROVED** [review]

**Pull request merged**